src/styles/blocks/card.scss

```diff
@@ -0,0 +1,101 @@
+.card {
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #{$card-border-color};
+  border-radius: 5px;
+  width: 200px;
+
+  &__imac {
+    display: block;
+    margin: 0 auto 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product {
+    &-name {
+      margin-bottom: 4px;
+
+      color: #{$text-default-color};
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    &-code {
+      color: #{$text-subheading-color};
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      margin-bottom: 16px;
+    }
+
+    &-quality {
+      display: flex;
+      width: 100%;
+      justify-content: space-between;
+      align-items: flex-end;
+      margin-bottom: 24px;
+    }
+
+    &-reviews {
+      margin-left: auto;
+      color: #{$text-default-color};
+      text-align: right;
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 16px;
+    }
+
+    &-price {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      margin-bottom: 16px;
+
+      &-title {
+        font-size: 12px;
+        font-weight: 400;
+        color: #{$text-subheading-color};
+        line-height: 18px;
+      }
+
+      &-cost {
+        margin-left: auto;
+        color: #{$text-default-color};
+        text-align: right;
+        font-size: 16px;
+        font-weight: 700;
+        line-height: 18px;
+      }
+    }
+
+    &-count {
+      color: #{$text-default-color};
+    }
+
+    &-button {
+      display: flex;
+      justify-content: center;
+      align-items: center;
+      padding: 11px 0;
```

Comment:
why not just `height: 40px;` ?

Reply:
[Viktor-Kost](https://github.com/Viktor-Kost) [Jul 12, 2023](https://github.com/mate-academy/layout_product-cards/pull/3317#discussion_r1261589701)

it's better not hardcode height for elements with content inside. Just use padding: 11px 0;